### PR TITLE
Move to contentgrid

### PR DESCRIPTION
**Pull request by rschev**
The thunx dependency might still need to change in light of `0.4.1`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Content Cloud Gateway
 
-An API Gateway based on [Spring Cloud Gateway](https://spring.io/projects/spring-cloud-gateway) with integration with [Open Policy  Agent](https://www.openpolicyagent.org) for Content Cloud projects. You can find a prebuilt image at [content-cloud/gateway](https://hub.xenit.eu/harbor/projects/49/repositories/gateway)
+An API Gateway based on [Spring Cloud Gateway](https://spring.io/projects/spring-cloud-gateway) with integration with [Open Policy  Agent](https://www.openpolicyagent.org) for Content Cloud projects. You can find a prebuilt image at [contentgrid/gateway](https://cloudsmith.io/~xenit/repos/contentgrid/packages/detail/docker/gateway/)
```
Comment (vierbergenlars) [inline README.md:3]: This is a private package repository.

File: build.gradle [hunk 2]
```diff
@@ -61,8 +61,9 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-security'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 
-	implementation "eu.xenit.contentcloud.thunx:thunx-spring:${thunxVersion}"
-	implementation "eu.xenit.contentcloud.thunx:thunx-pdp-opa:${thunxVersion}"
+	implementation platform("com.contentgrid.thunx:thunx-bom:${thunxVersion}")
+	implementation "com.contentgrid.thunx:thunx-spring:${thunxVersion}"
+	implementation "com.contentgrid.thunx:thunx-pdp-opa:${thunxVersion}"
```
Comment (tgeens) [inline build.gradle:66]: don't specify versions when you use the bom

File: build.gradle [hunk 3]
```diff
@@ -16,8 +16,8 @@ repositories {
 	maven {
 		url 'https://oss.sonatype.org/content/repositories/snapshots'
 		content {
-			includeGroup 'eu.xenit.contentcloud'
-			includeGroup 'eu.xenit.contentcloud.thunx'
+			includeGroup 'com.contentgrid'
```
Comment (tgeens) [inline build.gradle:19]: there is nothing published anymore under the root, you can remove this line

File: README.md (hunk 1 above)
Comment (tgeens) [reply README.md:3]: True, we should just remove that reference ?

Comment (tgeens) [reply README.md:3]: Also: double space in `Open Policy  Agent` ?

**Review by tgeens — APPROVED** [review]

**Pull request merged**